spk/gnupg/src/installer.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+# Package
+PACKAGE="gnupg"
+DNAME="GnuPG"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+
+preinst ()
+{
+    exit 0
+}
+
+postinst ()
+{
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+
+    # Add symlink
+    ln -s ${INSTALL_DIR}/bin/gpg-agent /usr/syno/bin/gpg-agent
```

Comment:
We can't put symlinks into system locations. `/usr/local/bin` is as far as we can go.